src/Core/src/Core/IApplication.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+
+namespace Microsoft.Maui
+{
+	/// <summary>
+	/// Encapsulates the Application and its available services.
+	/// </summary>
+	public interface IApplication
+	{
+		/// <summary>
+		/// Gets a collection of application-scoped services.
+		/// </summary>
+		IServiceProvider? Services { get; }
+
+		/// <summary>
+		/// Create the application main window.
+		/// </summary>
+		/// <param name="state">The activation state of the application.</param>
```

Comment:
this is not really the state of the application but the state of the activation of the application

Reply:
Updated comment.